app/javascript/components/create-comment-form.vue

```diff
@@ -2,50 +2,80 @@
 import { ref } from 'vue';
 import type { FeedbackSession } from 'api/feedback-sessions';
 import api from 'api/comments';
+import AlertComponent from 'components/alert-component.vue';
 
 interface Props {
   feedbackSession: FeedbackSession;
 }
 
 const props = defineProps<Props>();
 const bodyRef = ref('');
+const alertVal = ref(false);
```

Comment:
trataría de usar nombres más descriptivos para las variables

en el caso de `bodyRef`, se puede saber que es una `ref` porque estamos usando typescript, entonces agregarle el "`Ref`" al nombre de la variable no aporta mucho. Podrías ponerle `commentBody` o algo parecido, para que se cache que contiene el texto del comentario.

para `alertVal`, te recomiendo seguir la convención de javascript de ponerle `isBla` a las variables booleanas. En este caso podría ser algo como `isShowingAlert`. [Aquí](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) hay un artículo que me gusta con recomendaciones

Reply:
Muy buen alcance! Muchas gracias por el artículo, dejé cambiadas las variables para que cumplan con las convenciones!